apps/estuary/test/unit/data_writer_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule Estuary.DataWriterTest do
+  use ExUnit.Case
+  use Placebo
+
+  alias Estuary.Datasets.DatasetSchema
+  alias Estuary.DataWriter
+  alias SmartCity.TestDataGenerator, as: TDG
+
+  test "should insert event to history table" do
+    allow(MockTable.write(any(), any()), return: :ok)
```

Comment:
if you're using placebo, you shouldn't need to be mocking the mock. the table writer should be testing it's ability to write to presto so your implementation of table writer should be able us Mox to mock the table writer module and assert that the right things were passed to the table writer `write` function here.